src/game/SpellEffects.cpp

```diff
@@ -2927,37 +2941,6 @@ void Spell::EffectSummonPet(SpellEffectIndex eff_idx)
         return;
```

Comment:
Possible memory leak. (See Pet* NewSummon = new Pet;) Please verify that there is a delete or an initialization of your NewSummon for every possible code path.